SwiftParseChat/Constants.swift

```diff
@@ -77,10 +77,14 @@ let NOTIFICATION_APP_STARTED			= "NCAppStarted"
 let NOTIFICATION_USER_LOGGED_IN			= "NCUserLoggedIn"
 let NOTIFICATION_USER_LOGGED_OUT		= "NCUserLoggedOut"
 
-/* Chat Settings */
+/* Group Settings */
```

Comment:
Not sure this is the best way to do things. FYI I kept this style of constants from an old repo, but it's not really the Swift way of doing things; would probably be a good place for a struct.